job-runner/chain/traverser.go

```diff
@@ -411,6 +411,24 @@ func (t *traverser) runJobs() {
 		go func(job proto.Job) {
 			jLogger := t.logger.WithFields(log.Fields{"job_id": job.Id, "sequence_id": job.SequenceId, "sequence_try": t.chain.SequenceTries(job.Id)})
 
+			// If this is sequence start job (which currently means sequenceId == job.Id),
+			// wait for duration of SequenceRetryWait, then increment sequence try count.
+			if t.chain.IsSequenceStartJob(job.Id) {
+				if t.chain.SequenceTries(job.Id) != 0 {
+					jLogger.Infof(fmt.Sprintf("waiting %s before retrying sequence_id %s", job.SequenceRetryWait, job.SequenceId))
```

Comment:
all log lines printed by `jLogger` actually already include the sequence id (look above where jLogger is created — it's given a bunch of fields), so you dont' need to include the id in the message